### ISSUE TITLE
Wrong description when multiple custom-data-modal are defined

### ISSUE THREAD

**Issue by guillaumecharbonnier** [igvteam/igv-webapp#258]
Dear IGV team,
Thank you very much for your awesome work. I have been toying recently with the IGV webapp and I think I have found a minor bug.
I am using the latest prebuilt package (https://igv.org/app-archive/igv-webapp.2.13.3.zip) and adding a custom-data-modal to the track will use the "Description" from the first custom-data-modal defined in the trackRegistry.
If I add my own custom-data-modal after the 4DN, I get the [4DN Description](https://github.com/igvteam/igv-webapp/blob/270d2e77b5176d1730024a912a53028c59748af0/resources/tracks/hg38_4dn.json#L4) for both custom-data-modal, while I get my own Description if I put my own custom-data-modal first in the trackRegistry.

**Comment by jrobinso** [comment]
Sounds like a bug.  Could you post your custom data modal configuration?

**Comment by guillaumecharbonnier** [comment]
```
{
  "label": "Neoenhancer Positive Controls",
  "type": "custom-data-modal",
  "description": "All files related to neoenhancer positive control samples. Can be used to select a subset of files to explore the results.",
  "columns":
  [
    "name",
    "format",
    "donor_id"
  ],
  "delimiter": "\t",
  "data": "resources/tables/neoenhancer_positive_controls.tsv"
}
```

And this description is applied to the 4DN tracks if its json is loaded after in the `trackRegistry.json`:
```
  "hg38": [
    "resources/tracks/hg38_annotations.json",
    "resources/tracks/hg38_encode.json",
    "resources/tracks/hg38_neoenhancer_positive_controls.json",
    "resources/tracks/hg38_4dn.json"
  ],
```


![image](https://user-images.githubusercontent.com/20519429/232857451-77ebe65d-d0a4-4fe2-b65d-7dcc6fd9b097.png)

**Comment by turner** [comment]
This is fixed with commit 55c53fdd2cd362099419780c5a75039620ff1942
